nima/http2/webserver/src/test/resources/META-INF/services/io.helidon.nima.webserver.spi.ServerConnectionSelectorProvider

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+io.helidon.nima.http2.webserver.TestConnectionSelectorProvider
```

Comment:
why can't we generate this?

Reply:
If you have something in mind, create an issue. I do not see the reason for this to be generated, or where we would get the information for it